packages/botonic-react/src/utils.js

```diff
@@ -35,3 +35,23 @@ export const staticAsset = path => {
     return path
   }
 }
+
+/**
+ * given an object and a property, returns if the property exists (recursively)
+ * @param  {Object} arg1 object to be checked
+ * @param  {String} arg2 string describing the property (for nested values, separate them with '.')
+ * @return {Boolean}     true if property exists, false otherwise
+ */
+export const getProperty = (obj, property) => {
+  if (!property) return false
+  let properties = property.split('.')
+  for (let i = 0; i < properties.length; i++) {
+    let prop = properties[i]
+    if (!obj || !obj.hasOwnProperty(prop)) {
+      return false
+    } else {
+      obj = obj[prop]
+    }
+  }
+  return obj
+}
```

Comment:
Is this used?

Reply:
yes, it's used inside the current "useTheme" function in order to retrieve a property of the theme if this exists. Also i'm using it in some places to know if a property (chained or not) exists inside an object, replacing the previous conditions we use. For example:

`if(theme.brand && theme.brand.image || theme.brandImage)` --> `getProperty(theme, 'brand.image') || getProperty(theme, 'brandImage')`